### ISSUE TITLE
Allow zooming x and y-axis independently

### ISSUE THREAD

**Issue by jorisvandenbossche** [conbench/conbench#335]
I was looking at the link we got on a PR that said there were some regressions (https://conbench.ursa.dev/compare/runs/dd0dae5a44ef4f7995f9173159f340e5...f17c9aa4fe1343a68ea89e73a877c3d5/). The timeline plot allows you to zoom in and out (the +/- on right side of the plot), but, that always zooms both x and y at the same time. While if you want to zoom in on a small period in time, you typically only want to zoom in on the x-axis, while leaving the y-axis as is (otherwise you don't see the actual changes anymore). Knowing bokeh a little bit, I am assuming that it does support that kind of zooming, but that this maybe needs to be enabled?

To illustrate what I mean:

![image](https://user-images.githubusercontent.com/1020496/166934265-a19759de-9313-4380-9690-a040789c4cb2.png)

Above is the start view of the plot. There is a regression and I want to zoom in on that. The plot below is then zoomed in, trying to see the commits around the time of the regression, but because it also zoomed the y axis, the other commits before and after actually fall off the plot:

![image](https://user-images.githubusercontent.com/1020496/166934291-a8bb0c60-fd40-475d-b577-141920ae2804.png)

**Comment by dianaclarke** [comment]
Does using just the slider under the plot give you what you're looking for?

<img width="1114" alt="Screen Shot 2022-05-05 at 8 43 12 AM" src="https://user-images.githubusercontent.com/878798/166949576-b75aaba9-c485-4773-8284-8dc138a33de5.png">
<img width="1156" alt="Screen Shot 2022-05-05 at 8 45 15 AM" src="https://user-images.githubusercontent.com/878798/166949585-052ca98b-d184-4910-b2ff-8b3294cb2b46.png">

**Comment by jorisvandenbossche** [comment]
Indeed, that's a good way to get more or less the view that I want. I looked over that possibility (although there is a sentence that says that you can do that .. ;))

Personally I think it would still be handy to have access to a "box zoom" (https://docs.bokeh.org/en/2.4.0/docs/user_guide/tools.html#boxzoomtool) to more easily zoom into exactly the area you want.

**Comment by jgehrcke** [comment]
Great discussion. Let's do some plot tweaking.

**Comment by jgehrcke** [comment]
Quick update. The slider was removed in https://github.com/conbench/conbench/pull/611. I have collected ideas around Bokeh tools for the history plots inhttps://github.com/conbench/conbench/issues/645, including @jorisvandenbossche's feedback above about the box zoom.
